### ISSUE TITLE
[Keyboard Navigation - Bot Webchat - Carousel response ]: After sending the message the focus should be land on the first interactive element.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4030]
User Experience:
If focus order is not correct and does not moves to first interactive element after sending the message then it might confuse the user about the functionality of the control.

Repro Steps:
1. Open URL: Web Chat: Full-featured bundle (microsoft.github.io)
2. Web chat bundle will open.
3. Navigate on the web chat bot and enter "carousel" keyword in Type your message box, then select send button.
4. Cards related to the keyword will appear, navigate on the cards.
5. Check and verify that step 3-4 are accessible and MAS compliant.

Actual Result:
After sending the message the keyboard focus is not landed on the first interactive element.

Expected Result:
After sending the message the focus should be land on the first interactive element.

MAS Reference:
MAS 2.4.3 - Focus Order

Test Environment:
OS: Win 10 21H1
Build: OS Build 19043.1147
Browser: Edge Chromium Version 94.0.972.2 (Official build) dev (64-bit)
#URL: Web Chat: Full-featured bundle (microsoft.github.io)

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev1;#HCL;#MAS2.4.3;#Benchmark;#Accessibility;#Win10-Edge(Chromium);#E2E-HCL-BotFrameworkWebChat-Aug21;

**Comment by compulim** [comment]
We tried to implement this a few months ago but it's technically impossible. We detailed in this bug, https://github.com/microsoft/BotFramework-WebChat/issues/3135.

When the message arrives asynchronously (i.e. could be a few seconds later). Changing the focus asynchronously would interrupt what the screen reader is currently narrating.

Not to mention the focus order will be inconsistent when the bot send a plain text message vs. an interactive element. I.e. sometimes it will focus on the transcript, sometimes it will stay in the send box.

**Comment by v-shikch** [comment]
We have added this issue in our SME discussion queue. Will take action after getting feedback from SME.

**Comment by compulim** [comment]
Thanks @v-shikch. Look forward to you and SME feedbacks. Feel free to ping me on Teams for faster response.
